### PR TITLE
Update response-actions-windows-defender-advanced-threat-protection.md, issue #2943

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #2943, remove the "Note" section, as it doesn't make any sense since Windows Defender ATP’s endpoint detection & response is now available for all actual OS (Windows 7 or later for desktops, Windows Server 2012r2 or later for servers).
Source: 
1. [Microsoft finally brings Windows Defender ATP’s endpoint detection & response to Windows 7 and Windows 8.1](https://mspoweruser.com/microsoft-finally-brings-windows-defender-atps-endpoint-detection-response-to-windows-7-and-windows-8-1/)
2. [Protecting Windows Server with Windows Defender ATP](https://techcommunity.microsoft.com/t5/Windows-Defender-ATP/Protecting-Windows-Server-with-Windows-Defender-ATP/ba-p/267114#M128)

### CODE REVIEW

**Comment by mypil** [comment]
@mjcaparas - can you please assist to check, approve and merge the content for this PR based on issue #2943?

Thank you.

**Comment by mjcaparas** [comment]
@MaratMussabekov while the EDR capabilities are available on other OS (that is, other supported OS can be onboarded to the Windows Defender ATP platform), I believe the response actions that can be taken from Windows Defender Security Center (the dashboard) can only be done on Windows 10, version 1703 or later at this time.

Meaning, the actions from the Machine view will only available for the said versions of Windows 10. For example, you won't see the Isolate machine button on a server or Windows 7 machine.

Perhaps if there is a tweak that can be made to clarify this note, it  might be to state that machine related response actions are only available for machines on Windows 10, version 1703 or higher.

File related machine actions are likely OS agnostic.

**Comment by mjcaparas** [comment]
@MaratMussabekov I'm partially mistaken in my statement. Response actions are available on some servers but not all. It's available on Windows Server 1803, Windows Server 2019.

**Comment by mypil** [comment]
@MaratMussabekov - please see the comments from mjcaparas and kindly make the recommended changes to the PR. Thank you.

**Comment by MaratMussabekov** [comment]
Dear @mjcaparas , I have added Windows Servers, please review.
Thank you

File: windows/security/threat-protection/windows-defender-atp/response-actions-windows-defender-advanced-threat-protection.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ ms.date: 11/12/2017
 You can take response actions on machines and files to quickly respond to detected attacks so that you can contain or reduce and prevent further damage caused by malicious attackers in your organization.
 
 >[!NOTE]
-> These response actions are only available for machines on Windows 10, version  1703 or higher.
+> These response actions are only available for machines on Windows 10 (version  1703 or higher), Windows Server 1803 and Windows Server 2019.
```
Comment (mypil) [inline windows/security/threat-protection/windows-defender-atp/response-actions-windows-defender-advanced-threat-protection.md]:
```suggestion
> The machine related response actions are only available for machines on Windows 10 (version 1703 or higher), Windows Server, version 1803 and Windows Server 2019.
```

**Comment by MaratMussabekov** [comment]
Dear @mjcaparas , @mypil 
your changes were applied, thank you!

**Comment by mypil** [comment]
@mjcaparas - kindly assist to check and merge the content of this PR. Thank you.

**Pull request merged**